src/panels/lovelace/editor/types.ts

```diff
@@ -43,6 +43,8 @@ export interface EditorTarget extends EventTarget {
   configValue?: string;
   type?: InputType;
   config: ActionConfig;
+  key?: string;
```

Comment:
it feels like we're just adding all types to 1 interface. Shouldn't this be different interfaces?